src/colour/govuk-blue/index.md

```diff
@@ -78,7 +78,7 @@ Indicative examples for illustrative purposes only.
 
 <div class="img-border flex-center">
 
-![TODO](./youtube-profile.png)
+![Screenshot of YouTube profile on mobile, showing blue and teal used in the banner and social profile image.](./youtube-profile.png)
```

Comment:
This is repeating the blue and missing the white.

Reply:
Updated to remove the repetition of blue.

>Screenshot of YouTube profile on mobile, showing blue and teal used in the banner image.